packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+
+export enum AccordionHeadingLevel {
```

Comment:
An enum seems like overkill here and a bit of a disconnect  because you're forced to use the "h" before the level to satisfy language requirements.

Thoughts on just typing this as `1 | 2 | 3 | 4 | 5 | 6`?

Reply:
An enum member cannot have a numeric name. So I added the h in front. Is there a way around this?

Reply:
NM, I understand the suggestion now ;)